src/ORM/Connect/MySQLSchemaManager.php

```diff
@@ -39,6 +39,10 @@ public function createTable($table, $fields = null, $indexes = null, $options =
         }
         if ($indexes) {
             foreach ($indexes as $k => $v) {
+                // force MyISAM if we have a fulltext index
+                if ($v['type'] === 'fulltext') {
+                    $addOptions = 'ENGINE=MyISAM';
```

Comment:
Much nicer. Can we remove the `create_tabel` config from fulltextsearchable then?